clients/java/src/main/java/io/camunda/client/CamundaClient.java

```diff
@@ -1168,6 +1171,57 @@ ActivateAdHocSubProcessActivitiesCommandStep1 newActivateAdHocSubProcessActiviti
    */
   DeleteRoleCommandStep1 newDeleteRoleCommand(String roleId);
 
+  /**
+   * Command to assign a role to a tenant.
+   *
+   * <pre>
+   * camundaClient
+   *  .newAssignRoleToTenantCommand("tenantId")
+   *  .roleId("role-id")
```

Comment:
```suggestion
   *  .roleId("roleId")
```

🔧 Let's be consistent with the javadoc